my-app/src/Pages/ErrorPage.js

```diff
@@ -1,11 +1,13 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
 
 class Error extends Component {
   render() {
     return (
       <div>
         <div>
-          <h2>Error</h2>
+          <h2>404 Not Found</h2>
+          <Link to="/departments">Go to Home</Link>
```

Comment:
Use path as constant
```suggestion
          <Link to={DEPARTMENTS}>Go to Home</Link>
```